build/build.go

```diff
@@ -464,7 +464,6 @@ func NewSession(options *Options) (*Session, error) {
 		GOPATH:        options.GOPATH,
 		BuildTags:     options.BuildTags,
 		Minify:        options.Minify,
-		TestedPackage: options.TestedPackage,
```

Comment:
One thing that's not clear to me is if we should share a cache between `gopherjs test` and `gopherjs build`.

For example, let's say `foo` depends on `shared`, and you run `gopherjs test foo/...`. This will create a cache for `fake_dep/shared`. If you run a subsequent `gopherjs build`, is it okay to reuse the cache for `fake_dep/shared`?

I expect this would be fine as long as the rest of the `BuildCache` args are the same.